tests/components/emoncms/conftest.py

```diff
@@ -0,0 +1,47 @@
+"""Fixtures for emoncms integration tests."""
+
+from collections.abc import AsyncGenerator
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+UNITS = ["kWh", "Wh", "W", "V", "A", "VA", "°C", "°F", "K", "Hz", "hPa", ""]
+
+
+def get_feed(
+    number: int, unit: str = "W", value: int = 18.04, timestamp: int = 1665509570
+):
+    """Generate feed details."""
+    return {
+        "id": str(number),
+        "userid": "1",
+        "name": f"parameter {number}",
+        "tag": "tag",
+        "size": "35809224",
+        "unit": unit,
+        "time": timestamp,
+        "value": value,
+    }
+
+
+FEEDS = []
+for i, unit in enumerate(UNITS):
+    FEEDS.append(get_feed(i + 1, unit=unit))
```

Comment:
Can we make this a list expression?

`[{...dict...} for i, unit in enumerate(UNITS)]`

Reply:
done

Reply:
done